lib/roger_ui/roger.ex

```diff
@@ -0,0 +1,38 @@
+defmodule RogerUi.Roger do
+  @moduledoc """
+  Contract to Roger calls
+  """
+
+  @type queue :: %{
```

Comment:
Can you add `@typedoc` for the public types or make them private (`@typep`) if they should not be referenced outside this module?

Reply:
types declared privated